components/upload/UploadList.tsx

```diff
@@ -6,8 +6,12 @@ import Progress from '../progress';
 import classNames from 'classnames';
 import { UploadListProps, UploadFile, UploadListType } from './interface';
 
+const imageTypes: string[] = ['image', 'webp', 'png', 'svg', 'gif', 'jpg', 'jpeg', 'bmp'];
 // https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL
 const previewFile = (file: File, callback: Function) => {
+  if (file.type && !imageTypes.includes(file.type)) {
+    callback('');
+  }
```

Comment:
Can we add one more cache here to avoid preview file repeat?

Reply:
good idea ~

Reply:
https://github.com/ant-design/ant-design/blob/efe08a8b04d5221fb9c7e059ba7e7ae547c73ee6/components/upload/UploadList.tsx#L82，has avoided preview file repeat if `file.thumbUrl` existed.